test/karma/karma.config.js

```diff
@@ -64,9 +65,10 @@ module.exports = function(config) {
     ],
     browsers: browserStack
       ? Object.keys(browserStackLaunchers)
-      : Object.keys(localLaunchers),
+      // : Object.keys(localLaunchers),
+      : ['IE'],
 
-    singleRun: true, // set this to false to leave the browser open
+    singleRun: false, // set this to false to leave the browser open
```

Comment:
I couldn't get IE to successfully pass any tests on Karma's first run, so my workflow is to start up Karma, and refresh IE after it launches 🙁